.github/auto_assign_example.yml

```diff
@@ -0,0 +1,56 @@
+# Set to true to add reviewers to pull requests
```

Comment:
[IMO] There are more setting properties, and it is not clear which property is required, so I sugget to add a comment.

```suggestion
# Set to true to add reviewers to pull requests (required)
```